core/tracing/hooks.go

```diff
@@ -235,6 +235,10 @@ const (
 	// account within the same txn (captured at end of tx).
 	// Note it doesn't account for a self-destruct which appoints itself as recipient.
 	BalanceDecreaseSelfdestructBurn BalanceChangeReason = 14
+
+	// FIXME: V3_MERGE
+	// Optimism
+	BalanceIncreaseOptimismL1Cost BalanceChangeReason = 100
```

Comment:
These seem to be reasons for tracing and debugging. How should we name these and number these wrt to existing ones?